go-controller/pkg/node/gateway_udn.go

```diff
@@ -918,6 +924,9 @@ func (udng *UserDefinedNetworkGateway) doReconcile() error {
 		return fmt.Errorf("error while updating logical flow for UDN %s: %s", udng.GetNetworkName(), err)
 	}
 
+	if err := udng.updateAdvertisedUDNIsolationRules(isNetworkAdvertised); err != nil {
```

Comment:
so.. we don't need to do it while adding/deleting the network ? -> this reconcile takes care of add/updates and deletes?